build/tools/travis-ci.sh

```diff
@@ -54,7 +54,14 @@ case $wxTOOLSET in
         ;;
     *)
         echo 'Configuring...' && echo -en 'travis_fold:start:script.configure\\r'
-        ./configure --disable-optimise $wxCONFIGURE_FLAGS
+        ./configure --disable-optimise --disable-debug_info $wxCONFIGURE_FLAGS || rc=$?
+        if [ $rc != 0 ]; then
```

Comment:
`build/tools/travis-ci.sh: line 58: [: !=: unary operator expected`

Reply:
Sorry, could you please explain where do you see this? I must be blind, but I don't see anything wrong with this test...

Reply:
line 1305: [https://travis-ci.org/github/wxWidgets/wxWidgets/jobs/678182859](url)

`rc` is not set if configure return code is 0.

Reply:
Thanks, I've committed an ugly fix in 3fc119f564 (there must be a better way to do it, but somehow I don't see it at the moment...).